### ISSUE TITLE
[TreeView] ANDI does not read submodules correctly

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-react#1640]
When using ANDI to perform a test on the TreeView component, the sub-modules are not read correctly, this can be reproduced in the following example:

https://stackblitz.com/edit/react-ybhddy-uhextg?file=app%2Fmain.jsx

![image](https://github.com/telerik/kendo-react/assets/92358162/c6d9c9b1-d8fd-4847-94af-8b2efb7b7533)

Ticket ID: 1616040
